flink-shaded-jackson-parent/flink-shaded-jackson-2/src/main/resources/META-INF/NOTICE

```diff
@@ -6,11 +6,11 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- com.fasterxml.jackson.core:jackson-annotations:2.13.4
-- com.fasterxml.jackson.core:jackson-core:2.13.4
-- com.fasterxml.jackson.core:jackson-databind:2.13.4.2
-- com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.13.4
-- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.4
-- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.13.4
-- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.4
-- org.yaml:snakeyaml:1.31
+- com.fasterxml.jackson.core:jackson-annotations:2.14.2
+- com.fasterxml.jackson.core:jackson-core:2.14.2
+- com.fasterxml.jackson.core:jackson-databind:2.14.2
+- com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.14.2
+- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.14.2
+- com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.14.2
+- com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.14.2
+- org.yaml:snakeyaml:1.33
```

Comment:
Upgrading to 2.0 is not an option here?

Reply:
No, snakeyaml is a transitive dependency of Jackson and 2.0 is a breaking change

Reply:
Do you think that now that Jackson 2.16 has upgraded snakeyaml to 2.0 it would be possible to release a new flink-shaded-jackson with both upgrades ?

Reply:
I first want to complete the upgrade of Flink to the new dependencies before tackling another Flink shaded update.